cea/optimization/slave/cooling_resource_activation.py

```diff
@@ -144,268 +322,265 @@ def cooling_resource_activator(Q_thermal_req,
                 E_Trigen_NG_gen_W = np.float(eta_elec_interpol(NG_Trigen_req_W)) * NG_Trigen_req_W
 
             else:  # Only part of the demand can be delivered as 100% load achieved
+                # This case should never occur, since it means that another heat source would be required to run
+                # the absorption chiller at the required capacity.
+                print("WARNING: Gas turbine can not output enough heat to power the absorption chiller!")
```

Comment:
Have you considered to raise an error here, instead of just printing. Since is a case it should never occur...

Reply:
I slightly prefer leaving it at a printed warning since I don't want this to crash the optimisation if it were to occur in some odd case after all. And since any potential user of the CEA dashboard can't really change anything about the initialisation of the CHP in the settings, introducing an error here would also mean they could not run their case without modifying the code.

Reply:
Ah, makes sense. In this case I think there are some _raise warning_ options which doesn't stop the simulation. But I am not sure if it results in a practical difference compared to just printing it as you are doing.
Is there something else that the user can check (in terms of inputs) to fix this error? If so, it might be worth including.